### PR TITLE
Burner Additions

### PR DESCRIPTION
**Pull request by jasondegraw**
Add a burner quantity and control type. We still need to verify the control type enum values, but I'll go ahead and make the pull request.

### CODE REVIEW

File: BuildingSync.xsd [hunk 1]
```diff
@@ -14777,6 +14803,21 @@ Window spacing: the dimension between windows in a discrete window layout. (in.)
 			</xs:restriction>
 		</xs:simpleType>
 	</xs:element>
+	<xs:element name="BurnerControlType">
+		<xs:annotation>
+			<xs:documentation>Control type of burner, if applicable.</xs:documentation>
+		</xs:annotation>
+		<xs:simpleType>
+			<xs:restriction base="xs:string">
+				<xs:enumeration value="Full Modulation Set to Manual"/>
```
Comment (jasondegraw) [inline BuildingSync.xsd]: @supriyagoel Is this burner control type what we really want here? We're not too sure about this name.

File: BuildingSync.xsd [hunk 2]
```diff
@@ -14777,6 +14803,21 @@ Window spacing: the dimension between windows in a discrete window layout. (in.)
...
+				<xs:enumeration value="Full Modulation Set to Modulation"/>
```
Comment (jasondegraw) [inline BuildingSync.xsd]: @supriyagoel Same here, we're not so sure about this name either.

**Comment by jasondegraw** [comment]
@supriyagoel Will you have time to look at the burner-related naming in this PR? We could add you as a reviewer if that would help.

**Comment by supriyagoel** [comment]
Yes, I’ll do that, do add me as a reviewer.

Thanks

File: BuildingSync.xsd (hunk 2 above)
Comment (supriyagoel) [reply BuildingSync.xsd]: These options are from NYC LL87 audit data collection form. Based on literature I found online, there are 3 predominant type of controls-
1. On Off
2. High Fire/Low Fire (specified as High Low in BSXML)
3. Modulating

I believe the LL87 form provides 3 options for the modulating control (Step Modulation, Full Modulation- Set to Manual, Full Modulation- Set to Modulating)

BSXML could either use the options from LL87 or the 3 control types listed above (along with an option for 'Unknown')

**Comment by fieldkm** [comment]
Opening this back up, I'd be fine with 6 options total:

1. On Off
2. High Fire/Low Fire (specified as High Low in BSXML)
3. Step Modulation
4. Full Modulation- Set to Manual
5. Full Modulation- Set to Modulating
6. Unknown

Let me know what you all think, @jasondegraw @supriyagoel

**Comment by jasondegraw** [comment]
@fieldkm I'd be much happier if options 4 and 5 could be shortened and didn't include the hyphens. That's just asking for trouble with em dashes and spaces.

**Comment by fieldkm** [comment]
That makes sense, @jasondegraw .  How about:

On Off
High Low
Step Modulation
Full Modulation Manual
Full Modulation Automatic
Unknown

**Comment by jasondegraw** [comment]
@fieldkm That works for me

**Comment by fieldkm** [comment]
@jasondegraw apologies for having to re-do your original work.  The branch had conflicts with develop, so it was easier to match it to develop and then make the changes again.  Can you review to make sure it seems like everything is in order?

**Pull request merged**